samanthas_telegram_bot/auxil/escape_for_markdown.py

```diff
@@ -0,0 +1,25 @@
+def escape_for_markdown(str_: str) -> str:
+    escaped_str = str_
+    for char in (
```

Comment:
To avoid creating this tuple for every function call it's better to move it to constant

Reply:
Done